kubernetes/e2e_test/test_apps.py

```diff
@@ -56,28 +58,33 @@ def test_create_deployment(self):
         self.assertIsNotNone(resp)
 
         options = v1_delete_options.V1DeleteOptions()
-        resp = api.delete_namespaced_deployment(name, 'default', body=options)
+        resp = api.delete_namespaced_deployment(name, 'default', body={})
```

Comment:
why don't we keep the v1 DeleteOptions?

Reply:
It's optional - I kept it just in case. I think sometime in the past, body was not optional for the delete methods so that was necessary. Now it is not.